### DISCUSSION TITLE
Is there a way to disable the switch mode confirmation dialog popup?

### DISCUSSION THREAD

**Discussion by renanfranca** [redhat-developer/vscode-java#2452] [Q&A]
By typing switch mode on command palette and selecting it
![command-pallete](https://user-images.githubusercontent.com/6011616/168278602-11ba1921-978c-4fd2-a9d2-9b3cb60289a6.png)

The confirmation dialog shows up
![switch-standard-confirmation-dialog](https://user-images.githubusercontent.com/6011616/168283975-38150936-d17a-4938-b00c-24689f862cd9.png)

I used the mouse because I didn't find a shortcut to jump to the confirmation dialog options.

Can I disable the confirmation dialog by settings? Does anyone know the shortcut to the confirmation dialog?

**Comment by rgrunber** [comment]
It you look into File -> Preferences -> Keyboard Shortcuts, there is an option "Notifications : Focus Notification Toast". If I bind that to some key, and activate it after selecting the command, it focuses it and allows me to use tab to go through the options.

Another option is the "Notifications: Show Notifications" binding. This is also a command, so you could activate it from the command palette, but that seems a bit convoluted to simply gain focus.
